core/include/cubos/debug_render.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef CUBOS_DEBUG_RENDER_HPP
+#define CUBOS_DEBUG_RENDER_HPP
+
+#include <cubos/io/window.hpp>
+#include <glm/glm.hpp>
+
+namespace cubos
+{
+    class debug
+    {
+    public:
+        static void init(cubos::io::Window* window);
```

Comment:
Maybe instead of passing the window pass just the render device? I I don't think the debug renderer needs to know the window exists.

Reply:
It needs to know so it can know the current size of the framebuffer

Reply:
nvm, gonna remove it